pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -3,21 +3,23 @@
 import org.jenkinsci.plugins.workflow.libs.Library
 import org.folio.utilities.Logger
 import org.folio.rest_v2.Constants
+import org.folio.models.parameters.KarateTestsParameters
 import org.folio.models.parameters.CreateNamespaceParameters
 
+//TODO Switch branch before final merge
 //@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
-@Library('pipelines-shared-library@RANCHER-1002') _
+@Library('pipelines-shared-library@RANCHER-1001') _
```

Comment:
???

Reply:
Just for current testing, will be changed during final merge